src/page.ts

```diff
@@ -58,6 +58,8 @@ export interface PageDelegate {
   canScreenshotOutsideViewport(): boolean;
   resetViewport(): Promise<void>; // Only called if canScreenshotOutsideViewport() returns false.
   setBackgroundColor(color?: { r: number; g: number; b: number; a: number; }): Promise<void>;
+  startVideoRecording(options: types.VideoRecordingOptions): Promise<void>;
```

Comment:
Let's move this to the browser context to cover popups, wdyt?

Reply:
That's the ultimate plan. This requires generating file names, events that video started etc. For now I'd like to have basic functionality to provide some test coverage. Context level API can be added on top of it. Do you think we won't need page level start/stop functionality?

Reply:
As long as it is on delegate only - lgtm!